bin/lib/check_reqs.js

```diff
@@ -362,6 +362,36 @@ module.exports.run = function () {
     });
 };
 
+/** Checks if Windows SDK required to build the target_platform is present
+ * @param {String}  target_platorm        Target platform ('8.1' or '10.0')
+ * @return {Boolean}                      Whether the min SDK required to build the platform is present  
+ *
+ */ 
+module.exports.isWinSDKPresent = function (target_platform) {
+    return getInstalledWindowsSdks()
+    .then(function (installedSdks) {
+        var reqVersion = Version.tryParse(REQUIRED_VERSIONS[target_platform].windowssdk);
+        var hasSdkInstalled = installedSdks.some(function (installedSdk) {
```

Comment:
In `getInstalledWindowsSdks` If registry search fails, `fail` handler passes empty string below, as if it didn't get any output. Due to this regex search fails to find any SDKs and the last line returns an empty array
```js
     var installedSdks = [];
     // ... skipped ...
     .thenResolve(installedSdks);
```